cmd/cored/default.go

```diff
@@ -0,0 +1,43 @@
+//+build !protected_db
+
+// This configuration includes access to the reset endpoint.
+package main
+
+import (
+	"chain/core/config"
+	"chain/core/coreunsafe"
+	"chain/database/pg"
+	"chain/env"
+	"chain/log"
+	"context"
+	"fmt"
+	"os"
+)
+
+var (
+	reset = env.String("RESET", "")
+)
+
+func init() {
+	config.BuildConfig.ProtectedDB = false
```

Comment:
This makes sense. We should probably opt for config.go because the build constraints will become the AND of all the build tag negations. Setting one of the build tags should exclude this file.